tests/PositiveFloatString.test.ts

```diff
@@ -62,4 +75,9 @@ describe('PositiveFloatString', () => {
       expect(result._tag).toBe(expectedTag)
     })
   })
+  it('converts to a PositiveFloat', () => {
+    const numStr = '1.1'
+    if (!Guard.is(numStr)) throw new Error('Unexpected result')
+    expect(toPositiveFloat(numStr)).toBe(1.1)
```

Comment:
Just copying a comment I'd made on Discord so that it's part of the public record:

> It could be really cool to also provide an `Arbitrary` instance for these modules.. it would be expanding the scope quite a bit, and it would be really tricky to not include fast-check in the bundle.. but I think [OMITTED] had worked on this at some point. In any case, not something I think we should worry about for 1.0, but might be worth considering for v2